packages/mapviewer/tests/cypress/tests-e2e/search/search-results.cy.js

```diff
@@ -44,9 +45,49 @@ function checkDescendantOf(selector, message) {
         return $element
     }
 }
+const acceptedDelta = 0.1
+const checkLocation = (expected, result) => {
+    expect(result).to.be.an('Array')
+    expect(result.length).to.eq(2)
+    expect(result[0]).to.approximately(expected[0], acceptedDelta)
+    expect(result[1]).to.approximately(expected[1], acceptedDelta)
+}
+
+function testQueryPositionCrosshairStore({
+    searchQuery,
+    expectedCenter,
+    expectedPinnedLocation,
+    expectedCrosshair = null,
+    expectedCrosshairPosition = null,
+}) {
+    // check the query
+    cy.readStoreValue('state.search.query').should('eq', searchQuery)
+    // check the center of the map
+    cy.readStoreValue('state.position.center').should((positionCenter) =>
+        checkLocation(expectedCenter, positionCenter)
+    )
+    // check the location of pinnedLocation
+    cy.readStoreValue('state.map.pinnedLocation').should((pinnedLocation) =>
+        checkLocation(expectedPinnedLocation, pinnedLocation)
+    )
+    // check the crosshair
+    if (expectedCrosshair !== null) {
+        cy.readStoreValue('state.position.crossHair').should('eq', expectedCrosshair)
+    } else {
+        cy.readStoreValue('state.position.crossHair').should('be.null')
+    }
```

Comment:
unless I am mistaken, I think if you remove the if-else statement and only do : 
`cy.readStoreValue('state'.position.crosshair').should('eq', expectedCrosshair)`
it should work both when its value is null and when it is not.

Reply:
ah yes, it's correct. Thanks!